build-protos.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ -f mocs ]; then
+    rm -rf mocs
+fi
+
+if [ -d protobuf ]; then
+    rm -rf protobuf
+fi
+
+mkdir -p  protobuf mocks/mock_messaging
+
+# Generate protos
+./protogen
+
+# Update import paths on generated protos
+repourl=github.com/hyperledger/sawtooth-sdk-go
+grep -rl '"protobuf/' protobuf/ | while IFS= read -r file; do
+    sed -i "s|\"protobuf/|\"${repourl}/protobuf/|" "$file"
+done
+
+(
+cd messaging || exit
+mockgen -source connection.go >../mocks/mock_messaging/connection.go
+)
```

Comment:
Since this file is doing more than just build protos, it should be called something more general like `gen.sh`. Also, is there a reason you moved these commands out of `gen.go` and into this script? My concern is the script evolving to do arbitrary build-like things instead of just code-gen. Naming it `gen.sh` maybe helps a little to clarify the link.

Reply:
I did not find a way to do multiline scripts in the go generate syntax causing really long one liners, and working with variables got really strange as you would have to do things like $DOLLAR{var} to get $var and it got really ugly.